src/utils/web3.js

```diff
@@ -155,8 +231,9 @@ export const getProviderGasPrice = () => {
 
 // CONNECT TO CONTRACT WITH PROVIDER & SIGNER IF AVAILABLE
 export const getTokenContract = (tokenAddress) => {
+  const abi = getAbis().erc20
```

Comment:
are you sure that always you will get the prop `erc20` otherwise this will fail

Reply:
If getAbis() fails, it should load the default as it says:

If('testnet') {X}

Else{default object}

In both cases all props are covered by a const object list unless someone changes them, in which case we want it to break so that we know it needs fixing 

I can however change it so that we use a '?'

Reply:
ok if all these props are already defined then it is ok the issue would the possibility to get undefined from the function